contributors.md

```diff
@@ -1,49 +1,58 @@
+For help making changes, join our `#development` channel in [Slack](https://slack.dataform.co), or [email us](mailto:opensource@dataform.co) directly.
 
 ## Requirements
 
 [Bazel](https://bazel.build) - Bazel is a build system, and this is the only dependency you need to build and run the entire project.
 
-### Building
+## Getting Started
 
-To check the project builds succesfully, run the following command:
+First :fork_and_knife: [fork this repository](https://github.com/dataform-co/dataform/fork), clone it to your desktop, and navigate within.
 
-```bash
-bazel build ...
-```
+### Run the CLI
 
-_Note: If you are running Bazel on a **Mac**, `bazel build ...` may fail with a `Too many open files in system` error. This is [due to a limitation](https://github.com/angular/angular-bazel-example/issues/178) on the default maximum open file descriptors. You can increase the limit by running `sudo sysctl -w kern.maxfiles=<LARGE_NUMBER>` (we use `65536`)._
+Print out the default help information:
 
-### Test
+```bash
+./scripts/run help
+```
 
-To test the project, run the following command:
+Create a new `redshift` toy project:
 
 ```bash
-bazel test ...
+mkdir /tmp/test_project
+./scripts/run init redshift /tmp/test_project
 ```
 
-_Note: Some integration tests may fail due to missing credentials. If you need to run these tests, reach out to the team on [Slack](https://slack.dataform.co)._
+More commands can be found by substituting `dataform` with `./scripts/run` in the [CLI documentation](https://docs.dataform.co/guides/command-line-interface).
 
-### Run the CLI
+### Test
 
-Print out the default help information:
+To test the Dataform project, run the following command:
 
 ```bash
-./scripts/run help
+bazel test -- tests/... -tests/integration/...
 ```
 
-Create a new `redshift` project in a temp directory:
+This runs all tests excluding the integration tests. If you need to run integration tests, please [get in touch](mailto:opensource@dataform.co) with the team.
+
+_Note: A `java failed` error suggests that Java needs to be installed._
+
+### Building
+
+Running the CLI will build the required components. To build other components, for example the api, use:
 
 ```bash
-./scripts/run init redshift /tmp/test_project
+bazel build api
 ```
 
-See the [CLI documentation](https://docs.dataform.co/guides/command-line-interface/) for more commands.
+_Note: Building the entire project with `Bazel build ...` will not work unless you have credentials provided by the team. A workaround to this is to create an empty `tools/stackdriver-github-bridge/env.yaml` file._
 
-## Run the documentation site
+_Note: If you are running Bazel on a **Mac**, `bazel build ...` may fail with a `Too many open files in system` error. This is [due to a limitation](https://github.com/angular/angular-bazel-example/issues/178) on the default maximum open file descriptors. You can increase the limit by running `sudo sysctl -w kern.maxfiles=<LARGE_NUMBER>` (we use `65536`)._
```

Comment:
IMO this should still come early in the doc, since `/scripts/run` already does a very large build.

Reply:
Have moved upwards and changed message a tad